assets/src/styles/main.css

```diff
@@ -7,12 +7,12 @@
   .sm\:prose {
     color: theme("colors.gray.800");
 
-    .anchor {
+    & .anchor {
```

Comment:
What does the `&` do?

Reply:
CSS Nesting: https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector